res/css/structures/_TagPanel.scss

```diff
@@ -65,7 +65,7 @@ limitations under the License.
     align-items: center;
     margin-top: 5px;
```

Comment:
It looks like this `margin-top` is actually causing the top and bottom of the community panel to have different amounts of spacing. What do you think about removing this `margin-top` and reverting back to `height: 100%`?

Reply:
Sounds good to me, I changed my code to match your suggestion :+1:
As to why this `margin` property is there in the first place: Unfortunately digging into Git I can't find out who added this style. The only previous commit is 79d3cca6e9a2769239220d6eb057b176535fc4c3, where a lot of sass files where pasted into the repository.

Reply:
Right, it looks it appeared as part of a larger experiment with a new design.